<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>511</width>
     <height>51</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;交通路标识别系统&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>191</width>
     <height>431</height>
    </rect>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="commandLinkButton_photo">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>186</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>开启摄像设备识别</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="commandLinkButton_">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>170</y>
     <width>186</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>个人主页</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_collect">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>260</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Adobe 黑体 Std R&quot;;</string>
   </property>
   <property name="text">
    <string>采集信息</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_recognize">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>260</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Adobe 黑体 Std R&quot;;</string>
   </property>
   <property name="text">
    <string>开始识别</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>310</y>
     <width>311</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>340</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Adobe 黑体 Std R&quot;;</string>
   </property>
   <property name="text">
    <string>识别结果</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_yes">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>390</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Adobe 黑体 Std R&quot;;</string>
   </property>
   <property name="text">
    <string>准确</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_no">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>390</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Adobe 黑体 Std R&quot;;</string>
   </property>
   <property name="text">
    <string>不准确</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_exit">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
